theano/gpuarray/tests/test_GpuMaxAndArgmax.py

```diff
@@ -0,0 +1,139 @@
+from unittest import TestCase
+from theano.tests import unittest_tools as utt
+import numpy as np
+import theano
+import theano.tensor as T
+
```

Comment:
import .config to have the gpuarray test setup (and protection against running when there is no GPU).